src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -84,7 +84,7 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
 const Small: React.FC<ChannelImageBadgeProps> = ({type}) => {
   if (type === 'COMMUNITY') {
     return (
-      <View style={[styles.badgeContainer, styles.containersigned_primary]}>
+      <View style={[styles.badgeContainer, styles.containersigned_secondary]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `containersigned_primary` to `containersigned_secondary` alters the background color of the 'COMMUNITY' badge. Ensure this color change aligns with the overall design and accessibility guidelines, particularly in terms of contrast ratios for readability.

```diff
- <View style={[styles.badgeContainer, styles.containersigned_primary]}>
+ <View style={[styles.badgeContainer, styles.containersigned_secondary]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->